code/src/NCModel/Features.ts

```diff
@@ -113,199 +112,14 @@ export abstract class NcWorker extends NcObject
     }
 }
 
-export abstract class NcSignalWorker extends NcWorker
+export class GainControl extends NcWorker
```

Comment:
Given this is now a "vendor specific" class, similar to DemoControl:

- Could they be rolled into the same class?
- Could this renamed to be called DemoGainControl?
- Or even, given it's actually a "vendor specific" class, renamed to VendorSpecificGainControl - a bit verbose, but explicit

Reply:
We could altough traditionally we have identified "standard" control classes by prefixing them with "Nc". Anything would be a vendor specific class.

Reply:
True, but GainControl and DemoGain should be at least consistently named in this code base, so GainControl and Gain or DemoGainControl and DemoGain

Reply:
Do you see the Demo control class as related to the gain functionality. My original intention for that was to be a Demo vendor specific worker which has example properties of a wide range of types (string, boolean, number, enum etc.). In my mind their names would be GainControl and Demo (or DemoWorker) and because they do not start with Nc they are not "standard" classes hence they have to be vendor specific ones.

Reply:
I guess it might be the term "Demo" that's confusing - I was taking the Demo to mean it was an example of a vendor specific class. But I understand the distinction you're making.  "Example" might be less confusing than "Demo" in this context.

Reply:
I understand. I will change it to something like "ExampleWorker".